src/sweeper/sweepers/duplicates.py

```diff
@@ -29,7 +29,7 @@ def sweep(self):
         with arcpy.EnvManager(workspace=self.workspace):
             description = arcpy.da.Describe(self.table_name)
             print(f'Working on Duplicates for: {self.table_name}')
-            if description['dataType'] == 'Table':
+            if description['dataType'].casefold() == 'table':
```

Comment:
There were other string comparisons that should also be casefolded. Please search for those and update them as well.